recipe/conda_build_config.yaml

```diff
@@ -200,12 +206,12 @@ vc:
   - 14
   - 14
 vlfeat:
-  - 0.9
-xerces_c:
-  - 3
+  - 0.9.20
+xerces-c:
```

Comment:
Not sure if having a `-` in the name is problematic.

Reply:
it is. `_` and `-` are equivalent in the config file

Reply:
I mean `_` in the config file and package name with `-` are equivalent

Reply:
Reverted.